manifests/nexus.pp

```diff
@@ -0,0 +1,68 @@
+#
+# Download from Nexus using REST API
+# More info here: https://repository.sonatype.org/nexus-restlet1x-plugin/default/docs/path__artifact_maven_content.html
+#
+define archive::nexus (
+  $ensure       = present,
+  $checksum_type = 'md5',
+  $packaging    = 'jar',
+  $url          = undef,
+  $gav          = undef,
+  $repository   = undef,
+  $classifier   = undef,
+  $extension    = undef,
+  $user         = undef,
+  $owner        = undef,
+  $group        = undef,
+  $extract      = undef,
+  $extract_path = undef,
+  $creates      = undef,
+  $cleanup      = undef,
+) {
+
+  include archive::params
+
+  $artifact_info = split($gav, ':')
```

Comment:
This line is what's causing problems with the future parser.  `split()` doesn't know how to handle `undef`.